src/main/java/org/inferred/freebuilder/IgnoredByEquals.java

```diff
@@ -0,0 +1,15 @@
+package org.inferred.freebuilder;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * If present, do not include this property in the generated
```

Comment:
Just FYI, I will flesh this JavaDoc, plus the README, out (in traditional opinionated fashion) after this PR — would you like to review that change?

Reply:
Great. I don't mind. There's definitely a lot of warnings, proceed at your own risk to express.